core/db/default/spree/zones.rb

```diff
@@ -1,17 +1,12 @@
 eu_vat = Spree::Zone.create!(name: "EU_VAT", description: "Countries that make up the EU VAT zone.")
 north_america = Spree::Zone.create!(name: "North America", description: "USA + Canada")
-
-["Poland", "Finland", "Portugal", "Romania", "Germany", "France",
- "Slovakia", "Hungary", "Slovenia", "Ireland", "Austria", "Spain",
- "Italy", "Belgium", "Sweden", "Latvia", "Bulgaria", "United Kingdom",
- "Lithuania", "Cyprus", "Luxembourg", "Malta", "Denmark", "Netherlands",
- "Estonia"].
-each do |name|
-  eu_vat.zone_members.create!(zoneable: Spree::Country.find_by!(name: name))
+%w(PL FI PT RO DE FR SK HU SI IE AT ES IT BE SE LV BG GB LT CY LU MT DK NL EE)
+.each do |name|
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.